apps/expand.go

```diff
@@ -56,10 +60,13 @@ type Expand struct {
 	Post     ExpandLevel `json:"post,omitempty"`
 	RootPost ExpandLevel `json:"root_post,omitempty"`
 
-	// Team: all for model.team, summary for Id, DisplayName, Name, Description,
+	// Team: all for model.Team, summary for Id, DisplayName, Name, Description,
 	// Email, Type.
 	Team ExpandLevel `json:"team,omitempty"`
 
+	// TeamMember: expand model.TeamMember if TeamID and ActingUserID are set.
+	TeamMember ExpandLevel `json:"team_member,omitempty"`
```

Comment:
Could we ever made a case for a `TeamMember` expand for `User`? Should the team and channel expand for  `ActingUser` make it more clear that that it's about `ActingUser`, by e.g. being named `ActingTeamMember`?

Reply:
Going to merge as is, @hanzei if you feel strongly that "ActingUser" prefix is meaningful, I'll do another PR for that.